crates/task-impls/src/consensus/handlers.rs

```diff
@@ -137,11 +137,6 @@ pub async fn create_and_send_proposal<TYPES: NodeType, V: Versions>(
         proposed_leaf.view_number(),
     );
 
-    consensus
-        .write()
-        .await
-        .update_last_proposed_view(message.clone())?;
-
```

Comment:
Out of curiosity, what's the reason for removing this line? Was it not doing anything?

Reply:
I think it's just being moved to to being stored when we receive the proposal

Reply:
Yeah I just centralized where it happens instead of having it happen in two separate spots. Also, this would only get stored when the node sends a proposal, so no one else would have access to it. By moving it here, everyone can now store it.